setup.py

```diff
@@ -16,7 +16,7 @@
     url='https://github.com/pretenders/pretenders',
     packages=find_packages(),
     install_requires=[
-        'bottle', 'argparse'
+        'bottle', 'argparse', 'aiosmtpd'
```

Comment:
BTW shouldn't we add something like `python_requires='>=3.7.0',`?

Reply:
Good call